superset/views/core.py

```diff
@@ -1064,6 +1064,16 @@ def save_dash(  # pylint: disable=no-self-use
         dash = session.query(Dashboard).get(dashboard_id)
         check_ownership(dash, raise_if_false=True)
         data = json.loads(request.form["data"])
+        remote_last_modified_time = data.get("last_modified_time")
+        current_last_modified_time = dash.changed_on.replace(microsecond=0).timestamp()
+        # prevent mid-air collisions
+        if remote_last_modified_time < current_last_modified_time:
+            return json_error_response(
+                "This dashboard was changed recently. "
+                "Please reload dashboard to get latest version.",
+                412,
+            )
+
         DashboardDAO.set_dash_metadata(dash, data)
```

Comment:
Is this going to persist `last_modified_time` in Dashboard metadata? That's not what we want, is it?

Reply:
there is no direct/automatic mapping from data to Dashboard modal properties, so this `last_modified_time` will not be saved into dashboards table. But i remove it from `data` to avoid confusion.